CMakeLists.txt

```diff
@@ -334,6 +334,17 @@ endif()
 
 add_compile_options(${IMPORTANT_WARNINGS} ${ACCEPTABLE_WARNINGS} ${EXTRA_WARNINGS})
 
+# Sanatizer configuration
+set(SANITIZER "OFF" CACHE STRING "Enable clang sanitizer")
+
+set_property(CACHE SANITIZER PROPERTY STRINGS OFF address undefined)
+
+if (NOT ${SANITIZER} STREQUAL "OFF")
```

Comment:
OFF, NO, etc evaluate to false. Anything else including `address` and `undefined` evaluates to true. So it is enough to simply write `if (SANITIZER)`.

Reply:
Thanks for the suggestion, fixed.